samtranslator/model/api/api_generator.py

```diff
@@ -646,7 +706,21 @@ def to_cloudformation(self, redeploy_restapi_parameters, route53_record_set_grou
         permissions = self._construct_authorizer_lambda_permission()
         usage_plan = self._construct_usage_plan(rest_api_stage=stage)
 
-        return rest_api, deployment, stage, permissions, domain, basepath_mapping, route53, usage_plan
+        generated_resources: List[Any] = []
```

Comment:
Why `Any`? I'd rather have a verbose, well-typed tuple than no typing.

Reply:
The goal is to make it as a `List[Resource]` but that requires more refactoring. E.g. `permissions` is a list and `usage_plan` is a tuple so it's mixed type. So I use `Any` for now and will make it `List[Resource]` when the refactoring is done.

Reply:
Then is it `List[Union[Resource, List[Resource], Tuple[Resource]]]`?

Reply:
Good point! Added types to the list